Resources/skeleton/web/config.php

```diff
@@ -12,7 +12,7 @@
     exit('This script is only accessible from localhost.');
 }
 
-require_once dirname(__FILE__).'/../app/SymfonyRequirements.php';
+require_once __DIR__.'/../app/SymfonyRequirements.php';
```

Comment:
Isn't that checked via Composer? https://github.com/sensiolabs/SensioDistributionBundle/blob/master/composer.json#L14